inc/dashboard/filter.class.php

```diff
@@ -164,7 +167,7 @@ static function user_tech(string $value = ""): string {
       ]);
    }
 
-   static function dropdown(
+   static function itemDropdown(
```

Comment:
Method renamed because of a conflict with the implementation of the ancestor. Before  this PR, the class does not have any ancestor.

I dislike this change, but it is necessary. Maybe I can improve it by finding a better name.